app.js

```diff
@@ -14,12 +14,44 @@ app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
 
-// ...
+hbs.registerPartials(__dirname + "/views/partials")
 
 // Add the route handlers here:
 
 app.get('/', (req, res) => {
   res.render('index');
 });
+app.get('/beers', (req, res) => {
+  punkAPI.getBeers()
+    .then((beersData)=>{
+      console.log('Beer data fetched')
+      res.render('beers', {beersData});
+    })
+    .catch((err)=>{console.log(err)})
+});
```

Comment:
add some spacing between your routes (and code in general) for better readability